<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="bttn_BACK_MAIN">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>430</y>
      <width>171</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Main Menu</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>311</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="bttn_CAPTURE_ZERO">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Capture Zero</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Span</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tb_SPAN">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QPushButton" name="bttn_SET_SPAN">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>Set Span</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>301</width>
      <height>188</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Tare</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcd_TARE"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="bttn_SYSTEM_ZERO">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Set System Zero</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bttn_RESET_ZERO">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Reset Zero</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bttn_TARE_SCALE">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Tare Scale</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bttn_RESET_TARE">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Reset Tare</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>311</width>
      <height>76</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="bttn_STAGE_UP_SINGLE">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Stage Up Single Step</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bttn_STAGE_DOWN_SINGLE">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Stage Down Single Step</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>311</width>
      <height>112</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Steps</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="tb_NUMOFSTEPS"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="bttn_STAGE_UP_MULTI">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Stage Up Multiple</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bttn_STAGE_DOWN_MULTI">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Stage Down Multiple</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>330</y>
      <width>391</width>
      <height>84</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="bttn_GO_TOP">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Go Top</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_TOP_SET">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Not Set </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="bttn_GO_BOTTOM">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Go Bottom</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_BOTTOM_SET">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Not Set </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>240</y>
      <width>331</width>
      <height>70</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Weight</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcd_WEIGHT"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="bttn_POLL_SENSORS">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Poll Load Cell</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
